### PR TITLE
chore: fix type for country flags and hover titles

### PR DESCRIPTION
**Pull request by mbtools**
Avoid the following ts warning:

`Type 'FlagComponent' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.ts(2322)`

Adds country description on hover:

image

### CODE REVIEW

File: packages/plugins/ui-theme/src/i18n/crowdin/ui.json [hunk 1]
```diff
@@ -180,8 +180,9 @@
     "chineseTraditional": "Chinese Traditional"
   },
   "flag": {
-    "austria": "Austria",
```
Comment (juanpicado) [inline packages/plugins/ui-theme/src/i18n/crowdin/ui.json:183]: 😢 could you put Austria back? My friend @priscilawebdev created the UI from Austria 🇦🇹

Comment (mbtools) [reply packages/plugins/ui-theme/src/i18n/crowdin/ui.json:183]: There was no Austrian flag before (just Australia), so I thought it was a typo. I will of course add Austria!

**Comment by mbtools** [comment]
image

**Review by juanpicado — APPROVED** [review]
LGTM :) well deserved

**Pull request merged**